src/tests/JIT/Regression/JitBlue/Runtime_45250/Runtime_45250.ilproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
```

Comment:
Since we cannot represent this tail call pattern in C#, would it make sense to use an fsproj instead of (or in addition to) ilproj? JIT test infra already support F# (common props, dependencies etc. are set), e.g.: https://github.com/dotnet/runtime/blob/fae35941e16310d815460475810f069578e6e774/src/tests/JIT/Directed/tailcall/mutual_recursion.fsproj

Reply:
I could add the F# test, but I am not sure how much value the test would bring. We added many tests written in IL in the past that were designed to specifically expose some of the `tail. call` issues found during JIT stress testing. 

The original F# program in #45250 was failing during execution of some internal F# function (`invokeFast2`) and it would be  harder to debug that program and reason about the issue in comparison with the constructed minimal test repro.

@erozenfeld suggested to extend https://github.com/dotnet/runtime/blob/master/src/tests/JIT/Directed/tailcall/more_tailcalls.cs test suite and include the failing pattern in there, so I would say this should be our approach in order to increase test coverage for such tail call scenarios.